doc/operations/start_network.rst

```diff
@@ -90,17 +65,64 @@ To add a new node to an existing opening network, other nodes should be started
     join
     --network-cert-file /path/to/existing/network_certificate
     --target-rpc-address <another-ccf-node-address>
+    [--join-timer <join-retry-interval-ms>]
 
 The joining node takes the certificate of the existing network to join via ``--network-cert-file`` and initiates an enclave-to-enclave TLS connection to an existing node of the network as specified by ``--target-rpc-address``.
 
+A new node can only join an existing CCF network if its SGX quote is valid  [#remote_attestation]_. and runs an enclave application that is :ref:`trusted by the consortium <governance/common_member_operations:Updating Code Version>`.
+
 If the network has not yet been opened by members (see :ref:`governance/open_network:Opening the Network`), the joining node becomes part of the network immediately.
 
-A new node can only join an existing CCF network if its SGX quote is valid  [#remote_attestation]_. and runs an enclave application that is :ref:`trusted by the consortium <governance/common_member_operations:Updating Code Version>`.
+Otherwise, if the network has already been opened to users, members need to trust the joining node before it can become part of the network and participate in the consensus (see :ref:`governance/common_member_operations:Trusting a New Node`).
+
+The ``Pending`` joining node will automatically poll the service (interval configurable via ``--join-timer`` option) until the members have successfully transitioned the node to the ``Trusted`` state. It is only then that the joining node will transition to the ``PartOfNetwork`` state and start updating its ledger.
 
-If the network has already been opened to users, members need to trust the joining node before it can become part of the network (see :ref:`governance/common_member_operations:Trusting a New Node`).
+.. tip:: After the node has been trusted by members, operators should poll the ``/node/state`` endpoint on the newly added node until the ``{"state": "PartOfNetwork"}`` is reported. This status confirms that the replication of the ledger has started on this node.
```

Comment:
Should we talk about the difference between `PartOfNetwork` and "fully caught up" - ie, you may want to poll the node's commit index and only direct users to it once it has _finished_ replication?

Reply:
What does "finished" mean?

Reply:
Yeah, it's awkward to define "finished" here. If we decide to go the "[learner](https://etcd.io/docs/v3.3/learning/learner/#features-in-v35)" way at some point, we could have some thresholds to define when a node actually becomes part of the service (e.g. `N` transactions after it's seen its own addition to the store).

Reply:
I would prefer it not be mentioned until we have a proper definition for it.

Reply:
I think finished is up to the operator, but if we're clarifying the meaning of these states we should also clarify that a `PartOfNetwork` node may still be serving reads at `1.10`, despite your service being at `10.999999`?

Reply:
Sure, but that's always true, nodes can lag behind etc.

Reply:
Sure, so if you were monitoring all of your nodes and noticed it was lagging, you may want to remove it from your load balancer while you investigated. This lag is going to be most pronounced, and always present even in a well-behaved network, on a new joiner.

Reply:
If they join from a recent snapshot, it's going to be quite slight and rapidly plugged. Adding a reminder is fine, and the operator may have a QoS policy, but clients still need to be robust to followers lagging.